src/main/java/gov/usds/case_issues/db/model/User.java

```diff
@@ -0,0 +1,44 @@
+package gov.usds.case_issues.db.model;
+
+import java.time.ZonedDateTime;
+
+import javax.persistence.Entity;
+import javax.validation.constraints.NotNull;
+
+/**
+ * Metadata about the case issue data stored in this system
+ */
+@Entity
+public class User extends UpdatableEntity {
```

Comment:
Personally I think all tables should have the create/updated fields I think they are useful debugging information. With our current application code `updated_by` and `created_by` will always be null, `update_at` will equal `last_seen` and `created_at` will be the fist login. however I think the fields could still be useful:

For `updated_by`, I don't think that it is unreasonable that users might be updated by someone in the future (adding a profile photo? editing the display name? updated from a job similar to the upload service?).

For `created_at` it could be nice to know when the user first logged in.

For `updated_at` and our current application code it would be really interesting if it was ever not equal to `last_seen`. Might help debugging an issue with our oauth logic. and if we add user updatable fields it can become more useful

Not sure I can think of a case for `created_by` other than its nice to use `UpdatableEntity` and `WriteOnceEntity` for everything